### DISCUSSION TITLE
Error 1.0

### DISCUSSION THREAD

**Discussion by Airioo** [bnpr/Malt#307] [Q&A] [answered]
Hey ! While I register my file I have an error : 

Python: Traceback (most recent call last):
  File "C:\Users\utilisateur\AppData\Roaming\Blender Foundation\Blender\3.1\scripts\addons\BlenderMalt\MaltMaterial.py", line 136, in draw
    context.material.malt.draw_ui(layout, '.mesh.glsl', context.material.malt_parameters)
  File "C:\Users\utilisateur\AppData\Roaming\Blender Foundation\Blender\3.1\scripts\addons\BlenderMalt\MaltMaterial.py", line 63, in draw_ui
    if source_path != '' and source_path.endswith(extension) == False:
AttributeError: 'NoneType' object has no attribute 'endswith'

location: <unknown location>:-1

And also if I set material, after register all materials become black, could you help me please ?

**Comment by pragma37** [comment, marked as answer]
Please, open a bug report and make sure you include all the required info:
https://github.com/bnpr/Malt/issues/new/choose

Q&A is for "how to" questions.

**Reply by Airioo** [reply]
AH yeah sorry

**Reply by Airioo** [reply]
Problem solved with the new version
